app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
+  has_many :recipes
+  has_many :foods
```

Comment:
- [ ] Please make sure that you established the `association` correctly, it seems like your `association` is not correct. Kindly take a look again at the `ERD` diagram and implement the association properly. If you try to understand you will find that `food` itself is an independent entity regardless of user which is a part of `many` inventories or `recipes` through the join table.

![recipe_erd](https://github.com/DevKaranJ/recipes/assets/66476329/4a4b6196-bf8b-4b42-8f2c-10c6dfad05b6)